package.json

```diff
@@ -25,7 +25,7 @@
     "prepublishOnly": "npm run build"
   },
   "dependencies": {
-    "esquery": "^1.0.1"
+    "esquery-scope": "^1.1.0"
```

Comment:
Unfortunatly I had to keep the `esquery-scope` in order to be able to run the ci